### PR TITLE
Adds optionValuePath

### PR DESCRIPTION
**Pull request by jeremywrowe**
Adds optionValuePath to reflect option availability presented in readme.

### CODE REVIEW

File: tests/dummy/app/templates/blockless-single-option-value.hbs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  content=folks
+  selection=selectionValue
+  optionValuePath="content.cowboy"
+  optionLabelPath="content.name"
+  prompt="choose one"}}
+</p>
+<p>Cowboy? {{selectionValue}}</p>
+<p>All options:</p>
+<ul>
+{{#each folk in folks}}
```
Comment (Robdel12) [inline tests/dummy/app/templates/blockless-single-option-value.hbs]: Can this be updated to the new syntax? 

```handlebars
{{#each folks as |folk|}}
```

File: tests/dummy/app/controllers/blockless-single-option-value.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+import Folks from 'dummy/mixins/folks';
+
+export default Ember.ObjectController.extend(Folks, {
```
Comment (Robdel12) [inline tests/dummy/app/controllers/blockless-single-option-value.js]: Can this be `Ember.Controller `?

**Comment by Robdel12** [comment]
Have a typo in a filename :P `x-select-signle-blockless-option-value-test.js`

Other than those little thing, it looks good to me. @cowboyd any thoughts?

**Comment by jeremywrowe** [comment]
Thanks for catching these things. FYI. I didn't update the each helper and ObjectController reference on purpose. I wanted to mirror what was existing because I didn't want to break backwards compatibility of anything. :+1: for moving forward with new syntax :D

**Comment by cowboyd** [comment]
Looks excellent to me, thanks @jeremywrowe! I don't like the block-less form myself, but this is definitely a needed fix.

**Pull request merged**